src/main/scala/io/confluent/examples/streams/algebird/ProbabilisticCounter.scala

```diff
@@ -9,7 +9,7 @@ import org.apache.kafka.streams.processor.ProcessorContext
   * estimate.
   */
 class ProbabilisticCounter(val cmsStoreName: String)
-    extends Transformer[Array[Byte], String, KeyValue[String, Long]] {
```

Comment:
I'm not sure why, but these changes were necessary because IDEA reports scala compile errors, even when the code actually compiles. Even though this is really a bug in IDEA, I figured that it's preferable to present code that doesn't have apparent errors in such a common IDE.

Reply:
What does IDEA complain about?

Reply:
This should be necessary because, further down below, you made this change:

From

```scala
override def transform(key: Array[Byte], value: String): KeyValue[String, Long] = {
```
to 
```scala
override def transform(key: String, value: String): KeyValue[String, Long] = {
```
Note the type change for `key`.

Reply:
Yep, it was saying there was a type mismatch somewhere in the topology, which went away (for unknown reasons) when I switched the key to `String`. Pushing that change through resulted in the change to the counter.